docs/tutorials/goldenpath_series/gp_module_one.md

```diff
@@ -309,22 +298,24 @@ https://github.com/Unity-Technologies/com.unity.multiplayer.samples.poc/tree/fea
 ```csharp
         public override void OnNetworkSpawn()
         {
-            Move();
+            if (IsOwner)
+            {
+                Move();
+            }
         }
 ```
 </details>
 
-Any `MonoBehaviour` implementing `NetworkBehaviour` can override the  Netcode for GameObjects method `OnNetworkSpawn()`. This method is fired when the `NetworkObject` gets spawned and the networking is setup. We override `OnNetworkSpawn` since a client and a server will run different logic here. 
+Any `MonoBehaviour` implementing `NetworkBehaviour` can override the Netcode method `OnNetworkSpawn()`. This method is fired when the `NetworkObject` gets spawned and the networking is setup. We override `OnNetworkSpawn` since a client and a server will run different logic here. 
```

Comment:
This method is fired when the `NetworkObject` gets spawned.
(remove and the networking is setup).